modules/simple_forms_api/app/models/simple_forms_api/vba_21_4140.rb

```diff
@@ -30,6 +30,14 @@ def dob
       [trimmed_dob&.[](0..3), trimmed_dob&.[](4..5), trimmed_dob&.[](6..7)]
     end
 
+    def employed?
+      data['employers'].size.positive?
+    end
```

Comment:
This method will be used in a later PR when we're figuring out where to put signatures and dates.